sdk/src/main/java/software/amazon/awssdk/iot/iotjobs/IotJobsClient.java

```diff
@@ -68,15 +68,16 @@ public IotJobsClient(MqttClientConnection connection) {
 
     private Gson getGson() {
         GsonBuilder gson = new GsonBuilder();
+        gson.disableHtmlEscaping();
         gson.registerTypeAdapter(Timestamp.class, new Timestamp.Serializer());
         gson.registerTypeAdapter(Timestamp.class, new Timestamp.Deserializer());
         addTypeAdapters(gson);
         return gson.create();
     }
 
     private void addTypeAdapters(GsonBuilder gson) {
-        gson.registerTypeAdapter(JobStatus.class, new EnumSerializer<JobStatus>());
         gson.registerTypeAdapter(RejectedErrorCode.class, new EnumSerializer<RejectedErrorCode>());
+        gson.registerTypeAdapter(JobStatus.class, new EnumSerializer<JobStatus>());
```

Comment:
why did this change? we should use an ordered iterator, or ordered datastructure, when generating this code so we don't get arbitrary code-changes like this